### DISCUSSION TITLE
How can I remove this space from glyphs in my prompt?

### DISCUSSION THREAD

**Discussion by embeddedpenguin** [romkatv/powerlevel10k#2806] [General]
I've highlighted (in purple) the space to the left of os_icon.
![tmp](https://github.com/user-attachments/assets/cd9e450e-0b56-45a5-bda8-5830f620e878)

I've tried setting
`typeset -g POWERLEVEL9K_PROMPT_CHAR_LEFT_{LEFT,RIGHT}_WHITESPACE=false`. Setting this doesn't seem to affect anything.

**Comment by romkatv** [comment]
This looks like a bug in the terminal. It may go away if you switch to the recommended font.

**Reply by embeddedpenguin** [reply]
> This looks like a bug in the terminal. It may go away if you switch to the recommended font.

That did solve the issue, thanks!
